Inspicio/Controllers/ImagesController.cs

```diff
@@ -249,132 +282,63 @@ public class DataFromBody
             _context.Add(comment);
 
             await _context.SaveChangesAsync();
-			return Ok(1);
+            return Ok(1);
         }
-       
+        public enum States
```

Comment:
enums are usually singular, not plural - otherwise it gets strange when you have things like `void Foo(States state)` because state can only be a single value, not multiple. The plural tends to be for collections of things, e.g. `void Foo(List<State> states)`) 

[More](https://stackoverflow.com/a/1336012/1367)